File: AlumniDarli/app/Models/KontakUstadz.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class KontakUstadz extends Model
{
    use HasFactory;

    protected $fillable = [
        'nama',
        'jabatan',
        'bidang',
        'no_hp',
        'email',
        'foto'
    ];
}

File: Darli/app/src/main/java/com/example/darli/data/model/KontakUstadz.kt
package com.example.darli.data.model

data class KontakUstadz(
    val id: Int,
    val nama: String,
    val jabatan: String,
    val bidang: String?,
    val no_hp: String,
    val email: String?,
    val foto: String?
)

data class KontakUstadzResponse(
    val response_code: Int,
    val content: List<KontakUstadz>
)
